index.html

```diff
@@ -0,0 +1,177 @@
+    integrity="sha384-Zenh87qX5JnK2Jl0vWa8Ck2rdkQ2Bzep5IDxbcnCeuOxjzrPF/et3URy9Bv1WTRi" crossorigin="anonymous" />
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/js/bootstrap.bundle.min.js"
+    integrity="sha384-OERcA2EqjJCMA+/3y+gxIOqMEjwtxJY7qPCqsdltbNJuaOe923+mo//f6V8Qbsw3" crossorigin="anonymous"
+    defer></script>
+  <script src="https://kit.fontawesome.com/9d6e6aaa13.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+  <header class="page-header">
+    <section class="nav-bar">
+      <div class="page-logo">
+        <a href="#" class="navbar-brand">
+          <img width="60px" height="80px" src="img/main-icon.png" alt="icon">
+          World Snooker Championship 2022</a>
+      </div>
+      <nav class="nav-container">
+        <button id="menu-btn" type="button">
+          <i class="fa-solid fa-bars"></i>
+        </button>
+        <ul class="nav-menu">
+          <li class="active"><a href="#">Home</a></li>
+          <li>
+            <a href="about.html">About</a>
+          </li>
+          <li><a href="#program-section">Program</a></li>
+          <li><a href="#player-section">Players</a></li>
+          <li><a href="#partner-section">Partner</a></li>
+          <li class="nav-btn"><a href="#">Events</a></li>
+        </ul>
+      </nav>
+    </section>
+  </header>
+  <main>
+    <article class="headline-container" id="headline-section">
+      <section class="d-flex flex-column headline">
+        <h1>
+          <span class="greeting"> "Hello! Snooker World" </span>
+          World Snooker Championship 2022
+        </h1>
+        <p class="p-4 border border-3">
+          The <strong>World Snooker Championship 2022</strong> was organised by the World Snooker Tour
+          and sponsored by sports betting company Betfred. It featured a total prize fund of £2,395,000,
+          of which the winner received £500,000. The main stage of the tournament featured 32 players: the top 16
+          players from the snooker world rankings and an additional 16 players from the qualifying rounds.
+        </p>
+        <h2 class="event-date">2022.04.16(Sat) ~ 2022.05.02(Mon)</h2>
+        <p class="event-location">
+          @ Crucible Theatre in Sheffield, England
+        </p>
+      </section>
+    </article>
+    <article class="program-container d-flex flex-column align-items-center p-3" id="program-section">
+      <h2 class="section-heading">Main Program</h2>
+      <hr />
+      <section class="row gap-3 p-3 justify-content-center">
+        <section class="program-card col-md-2">
+          <div class="icon-container">
+            <i class="fa-solid fa-chalkboard-user"></i>
+          </div>
+          <h3 class="program-title">Lecture</h3>
+          <p class="p-1">
+            Listen to the snooker lessons taught by the most-experienced
+            international grandmasters.
+          </p>
+        </section>
```

Comment:
- [ ] Each of these cards are articles and as such should be wrapped in a `article` tag as this is the right semantic tag.